java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -71,4 +72,82 @@ public interface ServerManagementClusterCommands {
      *     value is the information of the sections requested for the node.
      */
     CompletableFuture<ClusterValue<String>> info(InfoOptions options, Route route);
+
+    /**
+     * Reads the configuration parameters of a running Redis server.<br>
+     * The command will be sent to a random node.
```

Comment:
@shohamazon I see that in python the documentation of config_get says that if it's called without routing it will be routed to all nodes, however in practice it is being sent to a random node. Can you fix that?

Reply:
same for nodejs

Reply:
its already fixed, https://github.com/aws/glide-for-redis/blob/ceefa0728a18b9620dc5cb45848482d2069aba8e/python/python/glide/async_commands/cluster_commands.py#L185

Reply:
#967